ui/app/AppLayouts/CommunitiesPortal/CommunitiesPortalLayout.qml

```diff
@@ -6,13 +6,16 @@ import StatusQ.Core 0.1
 import StatusQ.Core.Theme 0.1
 import StatusQ.Controls 0.1
 import StatusQ.Components 0.1
+import StatusQ.Popups 0.1
+import StatusQ.Popups.Dialog 0.1
 
 import utils 1.0
 import shared.popups 1.0
 
 import "controls"
 import "stores"
 import "popups"
+import "../../AppLayouts/Chat/panels/communities" // TODO correct import? or move somewhere else?
```

Comment:
Incorrect :P We must not use components from different AppLayouts. Please extract the needed component to the shared section.

Reply:
Thanks, this now lives in `ui/imports/shared/panels` (see first commit of this PR)

Reply:
Is this import still needed then? I guess you forgot to remove it :sweat_smile: